src/betterproto/__init__.py

```diff
@@ -739,6 +739,8 @@ def __bytes__(self) -> bytes:
                     output += _serialize_single(meta.number, TYPE_BYTES, buf)
                 else:
                     for item in value:
+                        if item is None:
```

Comment:
For performance it would be preferable to put the loop in a try block, and then in the except block iterate to find the None. It makes the error case a bit more complex and less efficient, but avoid adding the extra to the happy path.